README.md

```diff
@@ -219,8 +219,10 @@ Make sure that you replace "{repo-root-dir}" with the repository root directory
 ---- provider_settings.yaml ----<br>
 `provider_authority: "0.0.0.0:4010"`<br>
 `invehicle_digital_twin_uri: "http://0.0.0.0:5010"`<br><br>
-1. In the top window, run:<br><br>
+1. In the top window, run (config files will be loaded from the current working directory):<br><br>
 `./invehicle-digital-twin`<br>
+Use the `IBEJI_HOME` environment variable to load configuration files from a specific directory: <br>
```

Comment:
Your change here is for one specific sample.  You could discuss this environment variable before the details for each sample.  Alternatively, you could copy your change into each sample's details.

Reply:
I think a high level directive makes more sense and scales better. Let me move it upwards.